src/plugins/intel_gna/src/log/log.hpp

```diff
@@ -35,19 +35,19 @@ class GnaLog {
     /** Log level of particular log message */
     ov::log::Level message_level_ = ov::log::Level::NO;
 
+    static GnaLog& get_instance() {
+        static GnaLog log_obj;
+        return log_obj;
+    }
+
  public :
     GnaLog(const GnaLog&) = delete;
     void operator = (const GnaLog&) = delete;
 
-    GnaLog(ov::log::Level log_level) {
+    static void set_log_level(ov::log::Level log_level) {
```

Comment:
```suggestion
    static void init(ov::log::Level log_level) {
```

Theoretically the name is proper, but on the other hand this method should be invoked before any logging will be done.
Maybe we could call this init();

Reply:
That is true that "method should be invoked before any logging will be done".
Name `set_log_level` is longer and therefore more unique so for me it is easier to find all usages by simple grep (instead of intelisense which is always not working perfectly).
The same time longer name makes harder (I hope) to accidently insert `log::init()` call somewhere where it should not be, (and we should avoid such accidents as it affects the whole log mechanics).